test/Feature/HLSLLib/firstbithigh.16.test

```diff
@@ -0,0 +1,87 @@
+#--- source.hlsl
+
+StructuredBuffer<uint16_t4> In1 : register(t0);
+StructuredBuffer<int16_t4> In2 : register(t1);
+RWStructuredBuffer<uint32_t4> Out : register(u1);
```

Comment:
like we talked about offline make this u2 so the vulkan bindings are all different and they match the register numbres.